jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/comments/CaseCommentItemView.java

```diff
@@ -173,4 +195,67 @@ public void updateCommentPressingEnter(@ForEvent("keyup") final KeyboardEvent e)
         }
     }
 
+    @Override
+    public CaseCommentSummary getValue() {
+        return caseCommentSummary.getModel();
+    }
+
+    @Override
+    public void setValue(final CaseCommentSummary model) {
+        this.caseCommentSummary.setModel(model);
+        this.updateCommentText.setValue(model.getText());
+        setEditMode(editMode);
+        updateActions(model.getAuthor(),model.getId(),editMode);
+    }
+
+    protected void updateActions(final String author,final String id, final boolean editItem){
+        actionsItems.setInnerHTML("");
+        if (identity.getIdentifier().equals(author)) {
+            addAction( new CaseCommentsPresenter.CaseCommentAction() {
+                @Override
+                public String label() {
+                    return translationService.format(DELETE);
+                }
+
+                @Override
+                public void execute() {
+                    presenter.deleteCaseComment(id);
+                }
+            });
+
+            if (editItem) {
+                addUpdateCommentAction(new CaseCommentsPresenter.CaseCommentAction() {
+                    @Override
+                    public String label() {
+                        return translationService.format(SAVE);
+                    }
+
+                    @Override
+                    public void execute() {
+                        if (validateForm()) {
+                            presenter.updateCaseComment(getUpdatedComment(),id);
+                            setEditMode(!editItem);
+                        }
+                    }
+                });
+            }
+
+            addAction(new CaseCommentsPresenter.CaseCommentAction() {
+                @Override
+                public String label() {
+                    if (editItem) {
+                        return translationService.format(CANCEL);
+                    }
+                    return translationService.format(EDIT);
+                }
+
+                @Override
+                public void execute() {
+                   setEditMode(!editItem);
```

Comment:
With this new implementation it is possible to start editing multiple comments simultaneously, which per se doesn't have to be perceived as buggy behavior.
But when multiple comments are being edited and then one of the comments saved (using its Save option), the rest of currently edited comments are saved with old comment text. This might be perceived by user as loose behavior in comment editing functionality.